docs/docs/user/core-concepts/operation-mode.md

```diff
@@ -0,0 +1,136 @@
+# Operation Modes in KPOps
+
+KPOps supports three operation modes—`managed`, `manifest`, and `argo`. These modes determine how resources are managed and allow users to tailor their deployment strategy.
+
+- **Managed Mode** (default): KPOps uses Helm, and communicates with services like Kafka Rest Proxy, and Kafka Connect under the hood to manage the installation/(graceful) deletion of applications, creation/deletion of Kafka topics, creation/deletion of Connectors defined in the defined `pipeline.yaml`.
+- **Manifest Mode**: Focuses on generating Kubernetes manifests.
+- **Argo Mode**: Extends the functionality to include ArgoCD-specific hooks for certain operations, facilitating GitOps workflows with automated cleanup and reset tasks.
+
+---
+
+## Configuring Operation Modes
+
+### `manifest` Mode
+
+Enable `manifest` mode by setting the following in your configuration:
+
+```yaml
+operation_mode: manifest
+```
+
+### `argo` Mode
+
+Enable `argo` mode by setting the following in your configuration:
+
+```yaml
+operation_mode: argo
+```
```

Comment:
this is a bit verbose imo. It would be enough to link to the configuration page

Reply:
Re-written and linked the configuration page